app/views/concepts/_details.html.haml

```diff
@@ -2,47 +2,44 @@
   %div.p-1
     %div#change-request-notice
     - if change_requests_enabled?(@ontology.acronym)
-      %div#addProposalDropdownDiv{class: 'd-flex flex-row-reverse'}
-        %div.btn-group.dropstart
-          = button_tag('Add a proposal', id: 'addProposalDropdown', class: 'btn btn-sm btn-primary dropdown-toggle',
-                       type: 'button', 'data-bs-toggle': 'dropdown',  'aria-haspopup': 'true', 'aria-expanded': 'false')
-          %div.dropdown-menu{'aria-labelledby': 'addProposalDropdown'}
-            = link_to('Deepen class', '#', class: 'dropdown-item disabled')
-            = link_to('Delete class', '#', class: 'dropdown-item disabled')
-            = link_to('Obsolete class',
-                      change_requests_node_obsoletion_path(concept_id: @concept.id, concept_label: @concept.prefLabel,
-                                                           ont_acronym: @ontology.acronym),
-                      class: 'dropdown-item', 'data-turbo': 'true', 'data-turbo-stream': 'true', 'data-turbo-frame': '_top')
-            = link_to('Rename class',
-                      change_requests_node_rename_path(concept_id: @concept.id, concept_label: @concept.prefLabel,
-                                                       ont_acronym: @ontology.acronym),
-                      class: 'dropdown-item', 'data-turbo': 'true', 'data-turbo-stream': 'true','data-turbo-frame': '_top')
-      %div{'data-controller': 'change-requests'}
-        %div{id: 'addProposalFormDiv', 'data-change-requests-target': 'addProposalForm'}
-  = render ConceptDetailsComponent.new(id:'concept-details', acronym: @ontology.acronym, concept_id: @concept.id,
-                properties: @concept.properties,
-                top_keys: %w[description comment],
-                bottom_keys: %w[disjoint subclass is_a has_part]) do |c|
-
-    - c.header(stripped: true) do |t|
-      - t.add_row({th: t('ontology_details.concept.id')}, {td:link_to_with_actions(@concept.id, acronym: @ontology.acronym)})
-      - t.add_row({th: t('ontology_details.concept.preferred_name')}) do |h|
-        - h.td do
-          = display_in_multiple_languages(@concept.prefLabel)
-
-      - t.add_row({th: t('ontology_details.concept.synonyms')}) do |h|
-        - h.td do
-          %div.d-flex.justify-content-between
-            = display_in_multiple_languages(@concept.synonym, show_empty_alert: false, style_as_badge: true)
-            %div.synonym-change-request
-              = add_synonym_button
-              = remove_synonym_button
-
-
-      - unless @concept.definition.nil? || @concept.definition.empty?
-        - t.add_row({th: t('ontology_details.concept.definitions')}, {td: display_in_multiple_languages(@concept.definition)})
-
-      - if @concept.obsolete?
-        - t.add_row({th: t('ontology_details.concept.obsolete')}, {td: 'true'})
-
-      - t.add_row({th: t('ontology_details.concept.type')} , {td: @concept.type})
+      = render 'ontologies/add_proposal'
```

Comment:
@jvendetti here I moved this part to a partial, is it good with you ?

Reply:
I don't mind the move to a partial, but the `change-request-notice` div element is now duplicated in the DOM:

<img width="651" alt="Screenshot 2025-02-05 at 4 17 21 PM" src="https://github.com/user-attachments/assets/149cbb82-6edb-43a4-b727-7d7e92461998" />